lib/chai/core/assertions.js

```diff
@@ -1547,8 +1564,9 @@ module.exports = function (chai, _) {
   Assertion.addMethod('closeTo', closeTo);
   Assertion.addMethod('approximately', closeTo);
 
-  function isSubsetOf(subset, superset, cmp) {
-    return subset.every(function(elem) {
+  function isSubsetOf(subset, superset, cmp, ordered) {
+    return subset.every(function(elem, idx) {
+      if (ordered) return cmp ? cmp(elem, superset[idx]) : elem === superset[idx];
```

Comment:
I'll leave this here just to make other reviewer's job easier.
This line is related to the behaviour we talked about at [this post](https://github.com/chaijs/chai/issues/717#issuecomment-225432216).